src/reducers/comments.tsx

```diff
@@ -110,12 +112,24 @@ export const createCommentKey = ({ fileName, line }: CommentKeyParams) => {
 type CommonPayload = CommentKeyParams & { versionId: number };
 
 export const actions = {
+  abortLoadVersionComments: createAction(
+    'ABORT_FETCH_VERSION_COMMENTS',
```

Comment:
I don't understand why the action name is different than the constant.

Reply:
A refactoring mistake 😎